src/components/ContactCard/ContactCard.jsx

```diff
@@ -27,17 +32,32 @@ const ContactCard = ({ title, contact, renderHeader, renderBody }) => {
   const activeContactAccounts = getConnectedAccounts(contact)
 
   return (
-    <div>
+    <>
       {renderHeader(<ContactIdentity contact={contact} />)}
       {renderBody(
         <>
-          <ContactFields fields={normalizedFields} title={title} />
+          <Query query={allGroupsQuery}>
```

Comment:
J'ai un problème avec ça en fait, car ça veut dire que pour afficher un contact, on va faire 2 fois la requête pour récupérer tous les groupes : 

- Une fois dans le renderHeader via `ContactGroups` 
- Une fois dans le renderBody via la `Query` ci-dessus 

Dans l'ideal, il faudrait : 
- Récupérer la `allGroupsQuery` dans le <ContactCardModal> 
- Passer en props ces groups récupérés au Header et au Body

Reply:
@GoOz Let's focus on the UI part (see https://github.com/cozy/cozy-contacts/pull/483#discussion_r268779339), I'll take care of this one (see #484)